src/js/tech/html5.js

```diff
@@ -1332,7 +1329,7 @@ Html5.patchCanPlayType = function() {
 
   // Android 4.0 and above can play HLS to some extent but it reports being unable to do so
```

Comment:
I wonder if this is still the case or if this block can be removed.

Reply:
Yeah, this was for the browser before chrome.